SonarQube.Client/SonarQubeService.cs

```diff
@@ -54,7 +54,7 @@ public async Task<bool> HasOrganizations(CancellationToken token)
             return await Task.FromResult<bool>(hasOrganisations);
         }
 
-        public bool IsConnected { get; private set; }
+        public bool IsConnected => ServerInfo != null;
```

Comment:
There was no need to have an additional boolean, ServerInfo serves the same purpose.